packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -1474,5 +1474,45 @@ void main() {
 
       expect(factoryInvocationCount, 1);
     });
+
+    testWidgets('UiKitView has correct semantics', (WidgetTester tester) async {
+      final SemanticsHandle handle = tester.ensureSemantics();
+      final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
+      expect(currentViewId, greaterThanOrEqualTo(0));
+      final FakeIosPlatformViewsController viewsController =
```

Comment:
nit: no need to linebreak here, it makes the code harder to read.

Reply:
done